Formula/xorg-server.rb

```diff
@@ -0,0 +1,107 @@
+class XorgServer < Formula
+  desc "X Window System display server"
+  homepage "https://www.x.org"
+  url "https://www.x.org/releases/individual/xserver/xorg-server-21.1.4.tar.xz"
+  sha256 "5cc4be8ee47edb58d4a90e603a59d56b40291ad38371b0bd2471fc3cbee1c587"
+  license all_of: ["MIT", "APSL-2.0"]
+
+  depends_on "font-util"   => :build
+  depends_on "libxkbfile"  => :build
+  depends_on "meson"       => :build
+  depends_on "ninja"       => :build
+  depends_on "pkg-config"  => :build
+  depends_on "util-macros" => :build
+  depends_on "xorgproto"   => :build
+  depends_on "xtrans"      => :build
+
+  depends_on "libxfixes"
+  depends_on "libxfont2"
+  depends_on "mesa"
+  depends_on "pixman"
+  depends_on "xcb-util"
+  depends_on "xcb-util-image"
+  depends_on "xcb-util-keysyms"
+  depends_on "xcb-util-renderutil"
+  depends_on "xcb-util-wm"
+  depends_on "xkbcomp"
+  depends_on "xkeyboardconfig"
+
+  on_macos do
+    depends_on "libapplewm"
+  end
+
+  on_linux do
+    depends_on "dbus"
+    depends_on "libdrm"
+    depends_on "libepoxy"
+    depends_on "libxcvt"
+    depends_on "libxshmfence"
+    depends_on "nettle"
+    depends_on "systemd"
+  end
+
+  def install
+    # ChangeLog contains some non relocatable strings
+    rm "ChangeLog"
+    # set proper font dir
+    inreplace "meson.build", "fontutil_dep.get_pkgconfig_variable('fontrootdir')",
+              "'#{HOMEBREW_PREFIX}/share/fonts/X11'"
```

Comment:
What is `fontrootdir` currently set to in `fontutil`? I wonder if it's worth fixing there.

Reply:
`fontutil.pc`
```
prefix=/usr/local/Cellar/font-util/1.3.3
exec_prefix=${prefix}
libdir=${exec_prefix}/lib
datarootdir=${prefix}/share
datadir=${datarootdir}
fontrootdir=${datadir}/fonts/X11
mapdir=${fontrootdir}/util
 
Name: FontUtil
Description: Font utilities dirs
Version: 1.3.3
```
So the `fontrootdir` currently is `/usr/local/Cellar/font-util/1.3.3/fonts/X11`, the expected value is `/usr/local/fonts/X11`.
Formulae like `font-bh-100dpi` would link the fonts here.